_includes/head.html

```diff
@@ -90,7 +90,7 @@
 // export some helpful site-wide variables
 window.site = {
   baseurl: "{{ site.baseurl }}"{% unless site.public %},
-  access_token: "<!--# echo var="access_token" -->"{% endunless %}
+  access_token: '<!--# echo var="access_token" -->'{% endunless %}
```

Comment:
Ah, I think I see why you changed them. But the Server-Side Include replaces `var="access_token"`, so there's no issue with using `"` in both places. Don't bother changing it back, though.